### ISSUE TITLE
What formula do you use in back propagation？

### ISSUE THREAD

**Issue by Chrisable** [saurabhdash/GCN_Partitioning#2]
![image](https://user-images.githubusercontent.com/49330390/164868933-3021d74f-af9f-474d-bb6a-f7c964004bd3.png)

**Comment by saurabhdash** [comment]
All of them are required to calculate the gradients for backpropagation

**Comment by Chrisable** [comment]
Thanks for your reply.Excuse me, is custom_loss function the same as CutLoss?

**Comment by Chrisable** [comment]
When the loss function is added to the latter item, the data provided in the code will be very ineffective, and will other datasets encounter this situation? Is there any connection between good or bad performance and the dataset?
